### PR TITLE
Changes in Deployment steps for GA

### PR DESCRIPTION
**Pull request by sunasing**

### CODE REVIEW

File: windows/security/threat-protection/microsoft-defender-atp/ios-install.md [hunk 1]
```diff
@@ -20,62 +20,118 @@ ms.collection:
...
-> 
-> As with any pre-release solution, remember to exercise caution when determining the target population for your deployments.
-
-Defender for Endpoint for iOS is currently available as a preview app on TestFlight, Apple's beta testing platform. In GA, it will be available on the Apple App store.
-
-Deployment devices need to be enrolled on Intune Company portal. Refer to
-[Enroll your
-device](https://docs.microsoft.com/mem/intune/enrollment/ios-enroll) to
-learn more about Intune device enrollment
+This topic describes deploying Defender for Endpoint for iOS on Intune Company Portal enrolled devices. For more information about Intune device enrollment, see [Enroll your device](https://docs.microsoft.com/mem/intune/enrollment/ios-enroll)
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/ios-install.md:27]: Nitpick: sentence ending period dot.
```suggestion
This topic describes deploying Defender for Endpoint for iOS on Intune Company Portal enrolled devices. For more information about Intune device enrollment, see [Enroll your device](https://docs.microsoft.com/mem/intune/enrollment/ios-enroll).
```

File: windows/security/threat-protection/microsoft-defender-atp/ios-install.md [hunk 2]
```diff
@@ -20,62 +20,118 @@ ms.collection:
...
+
+1. On the Add app page, click on **Search the App Store** and type **Microsoft Defender ATP** in the search bar. On the search results section, click on *Microsoft Defender ATP* and click **Select**.
+
+1. Select **iOS 11.0** as the Minimum operating system. Review the rest of information about the app and click **Next**.
 
+1. In the *Assignments* section, go to the **Required** section and select **Add group**. You can then choose the user group(s) that you would like to target Defender for Endpoint for iOS app. Click **Select** and then **Next**.
 
-![Image of deployment steps](images/testflight-get.png)
+    >[!NOTE]
+    >The selected user group should consist of Intune enrolled users.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/ios-install.md:56]: May I suggest formatting consistency/MarkDown indent marker compatibility spacing?
```suggestion
    > [!NOTE]
    > The selected user group should consist of Intune enrolled users.
```

File: windows/security/threat-protection/microsoft-defender-atp/ios-install.md [hunk 3]
```diff
@@ -20,62 +20,118 @@ ms.collection:
...
+## Configure Microsoft Defender for Endpoint for Supervised Mode
+
+The Microsoft Defender for Endpoint for iOS app has specialized ability on supervised iOS/iPadOS devices given the increased management capabilities provided by the platform on these types of devices. To take advantage of these capabilities, Defender for Endpoint app needs to know if a device is in Supervised mode.
+
+### Configure Supervised Mode via Intune
+
+Intune allows you to configure the Defender for iOS app through a App Configuration policy.
+
+    >[!NOTE]
+    >This app configuration policy for supervised devices is applicable only to managed devices and should be targeted for all managed iOS devices as a best practice.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/ios-install.md:91]: (as above)
```suggestion
    > [!NOTE]
    > This app configuration policy for supervised devices is applicable only to managed devices and should be targeted for all managed iOS devices as a best practice.
```

**Review by illfated — APPROVED** [review]
Fresh new images, nice. Only cosmetic details to be edited from my perspective.

**Comment by illfated** [comment]
Oh. I noticed a couple of grammatical details too. May I add those as well?

File: windows/security/threat-protection/microsoft-defender-atp/ios-install.md [hunk 4]
```diff
@@ -20,62 +20,115 @@ ms.collection:
...
+Deploy Defender for Endpoint for iOS via Intune Company Portal.
+
+### Add iOS store app
+
+1. In [Microsoft Endpoint manager admin center](https://go.microsoft.com/fwlink/?linkid=2109431), go to **Apps** -> **iOS/iPadOS** -> **Add** -> **iOS store app** and click Select.
+
+    > [!div class="mx-imgBorder"]
+    ![Image of Microsoft Endpoint Manager Admin Center](images/ios-deploy-1.png)
+
+1. On the Add app page, click on **Search the App Store** and type **Microsoft Defender ATP** in the search bar. On the search results section, click on *Microsoft Defender ATP* and click **Select**.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/ios-install.md:49]: minor detail, "on" vs. "in"
```suggestion
1. On the Add app page, click on **Search the App Store** and type **Microsoft Defender ATP** in the search bar. In the search results section, click on *Microsoft Defender ATP* and click **Select**.
```

**Comment by sunasing** [comment]
> Oh. I noticed a couple of grammatical details too. May I add those as well?

Yes please

File: windows/security/threat-protection/microsoft-defender-atp/ios-install.md [hunk 5]
```diff
@@ -20,62 +20,118 @@ ms.collection:
...
+1. In the *Assignments* section, go to the **Required** section and select **Add group**. You can then choose the user group(s) that you would like to target Defender for Endpoint for iOS app. Click **Select** and then **Next**.
 
+    > [!NOTE]
+    > The selected user group should consist of Intune enrolled users.
 
-![Image of deployment steps](images/testflight-get.png)
+    > [!div class="mx-imgBorder"]
+    ![Image of Microsoft Endpoint Manager Admin Center](images/ios-deploy-2.png)
+
+1. In the *Review + Create* section, verify that all the information entered is correct and then select **Create**. In a few moments, the Defender for Endpoint app would be created successfully, and a notification would show up at the top-right corner of the page.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/ios-install.md:61]:
```suggestion
1. In the *Review + Create* section, verify that all the information entered is correct and then select **Create**. In a few moments, the Defender for Endpoint app should be created successfully, and a notification should show up at the top-right corner of the page.
```

File: windows/security/threat-protection/microsoft-defender-atp/ios-configure-features.md [hunk 6]
```diff
@@ -73,3 +65,7 @@ Defender for Endpoint for iOS enables admins to configure custom indicators on i
 ## Web Protection
 
 By default, Defender for Endpoint for iOS includes and enables the web protection feature. [Web protection](web-protection-overview.md) helps to secure devices against web threats and protect users from phishing attacks.
+
+## Report unsafe site
+
+Phishing websites impersonate trustworthy websites for the purpose of obtaining your personal or financial information. Visit [report unsafe site](https://www.microsoft.com/wdsi/filesubmission/exploitguard/networkprotection) if you want to report a website that could be phish.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/ios-configure-features.md:71]:
```suggestion
Phishing websites impersonate trustworthy websites for the purpose of obtaining your personal or financial information. Visit the [report unsafe site](https://www.microsoft.com/wdsi/filesubmission/exploitguard/networkprotection) page if you want to report a website that could be a phishing site.
```

File: windows/security/threat-protection/microsoft-defender-atp/ios-install.md [hunk 7]
```diff
@@ -20,62 +20,118 @@ ms.collection:
...
-3.  Upon successful onboarding, the device will start showing up on the Devices
-    list in Microsoft Defender Security Center.
+3. Upon successful onboarding, the device will start showing up on the Devices list in Microsoft Defender Security Center.
 
     > [!div class="mx-imgBorder"]
     > ![A screenshot of a cell phone Description automatically generated](images/e07f270419f7b1e5ee6744f8b38ddeaf.png)
 
+## Configure Microsoft Defender for Endpoint for Supervised Mode
+
+The Microsoft Defender for Endpoint for iOS app has specialized ability on supervised iOS/iPadOS devices given the increased management capabilities provided by the platform on these types of devices. To take advantage of these capabilities, Defender for Endpoint app needs to know if a device is in Supervised mode.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/ios-install.md:84]:
```suggestion
The Microsoft Defender for Endpoint for iOS app has specialized ability on supervised iOS/iPadOS devices, given the increased management capabilities provided by the platform on these types of devices. To take advantage of these capabilities, the Defender for Endpoint app needs to know if a device is in Supervised mode.
```

File: windows/security/threat-protection/microsoft-defender-atp/ios-install.md [hunk 8]
```diff
@@ -20,62 +20,118 @@ ms.collection:
...
+
+### Configure Supervised Mode via Intune
+
+Intune allows you to configure the Defender for iOS app through a App Configuration policy.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/ios-install.md:88]:
```suggestion
Intune allows you to configure the Defender for iOS app through an App Configuration policy.
```

File: windows/security/threat-protection/microsoft-defender-atp/ios-install.md [hunk 9]
```diff
@@ -20,62 +20,118 @@ ms.collection:
...
+    - Policy Name
+    - Platform: Select iOS/iPadOS
+    - Targeted app: Select **Microsoft Defender ATP** from the list
+
+    > [!div class="mx-imgBorder"]
+    ![Image of Microsoft Endpoint Manager Admin Center](images/ios-deploy-5.png)
+
+1. In the next screen, select **Use configuration designer** as the format. Specify the following property:
+    - Configuration Key: issupervised
+    - Value type: Sting
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/ios-install.md:108]:
```suggestion
    - Value type: String
```

File: windows/security/threat-protection/microsoft-defender-atp/ios-install.md [hunk 10]
```diff
@@ -20,62 +20,118 @@ ms.collection:
...
+    - Configuration Value: {{issupervised}}
+    
+    > [!div class="mx-imgBorder"]
+    ![Image of Microsoft Endpoint Manager Admin Center](images/ios-deploy-6.png)
+
+1. Click **Next** to open the **Scope tags** page. Scope tags are optional. Click **Next** to continue.
+
+1. On the **Assignments** page, select the groups that will receive this profile. For this scenario, it is best practice to target **All Devices**. For more information on assigning profiles, see [Assign user and device profiles](https://docs.microsoft.com/mem/intune/configuration/device-profile-assign).
+
+   When deploying to user groups, a user must sign-in on a device before the policy applies.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/ios-install.md:118]:
```suggestion
   When deploying to user groups, a user must sign in to a device before the policy applies.
```

File: windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios.md [hunk 11]
```diff
@@ -24,53 +24,52 @@ ms.topic: conceptual
...
-> This documentation is for a pre-release solution. The guidelines and the solution are subject to change between now and its general availability.
-> 
-> As with any pre-release solution, remember to exercise caution when determining the target population for your deployments.
-
-
-The public preview of Defender for Endpoint for iOS will offer protection
-against phishing and unsafe network connections from websites, emails, and apps.
-All alerts will be available through a single pane of glass in the Microsoft
-Defender Security Center. The portal gives security teams a centralized view of threats on
+**Microsoft Defender for Endpoint for iOS** will offer protection against phishing and unsafe network connections from websites, emails, and apps. All alerts will be available through a single pane of glass in the Microsoft Defender Security Center. The portal gives security teams a centralized view of threats on
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios.md:27]: Is this phrasing correct - "pane of glass" - describing pages or app sections in Microsoft Defender Security Center?

File: windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios.md [hunk 12]
```diff
@@ -24,53 +24,52 @@ ms.topic: conceptual
...
+- Access to the Microsoft Defender Security Center portal.
+
+    > [!NOTE]
+    > Microsoft Intune is the only supported Mobile Device Management (MDM) solution for deploying Microsoft Defender for Endpoint for iOS. Currently only enrolled devices are supported for enforcing Defender for Endpoint for iOS related device compliance policies in Intune.
 
--   Access to [Microsoft Endpoint Manager admin
-    center](https://go.microsoft.com/fwlink/?linkid=2109431), to deploy the app
-    to enrolled user groups in your organization
+- Access to [Microsoft Endpoint Manager admin
+    center](https://go.microsoft.com/fwlink/?linkid=2109431), to deploy the app to enrolled user groups in your organization
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios.md:52]: Line break/NewLine removal:
```suggestion
- Access to [Microsoft Endpoint Manager admin center](https://go.microsoft.com/fwlink/?linkid=2109431), to deploy the app to enrolled user groups in your organization
```

File: windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios.md (hunk 11 above)
Comment (sunasing) [reply windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios.md:27]: Yes thats how they refer it.

Thanks for other suggestions. Some of them were so silly on my part.

**Comment by illfated** [comment]
Just to be on the safe side, I suggest having a copy review by @JohanFreelancer9 (in case I missed some sentence details not obvious enough to me).

File: windows/security/threat-protection/microsoft-defender-atp/ios-configure-features.md [hunk 13]
```diff
@@ -27,14 +27,6 @@ ms.topic: conceptual
...
-> **PUBLIC PREVIEW EDITION**
-> 
-> This documentation is for a pre-release solution. The guidelines and the solution are subject to change between now and its general availability.
-> 
-> As with any pre-release solution, remember to exercise caution when determining the target population for your deployments.
-
-
 ## Configure compliance policy against jailbroken devices
 
 To protect corporate data from being accessed on jailbroken iOS devices, we recommend that you setup the following compliance policy on Intune.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/ios-configure-features.md:32]:
```suggestion
To protect corporate data from being accessed on jailbroken iOS devices, we recommend that you set up the following compliance policy on Intune.
```

File: windows/security/threat-protection/microsoft-defender-atp/ios-configure-features.md [hunk 14]
```diff
@@ -73,3 +65,7 @@ Defender for Endpoint for iOS enables admins to configure custom indicators on i
 ## Web Protection
 
 By default, Defender for Endpoint for iOS includes and enables the web protection feature. [Web protection](web-protection-overview.md) helps to secure devices against web threats and protect users from phishing attacks.
+
+## Report unsafe site
+
+Phishing websites impersonate trustworthy websites for the purpose of obtaining your personal or financial information. Visit the [report unsafe site](https://www.microsoft.com/wdsi/filesubmission/exploitguard/networkprotection) page if you want to report a website that could be a phishing site.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/ios-configure-features.md:71]:
```suggestion
Phishing websites impersonate trustworthy websites for the purpose of obtaining your personal or financial information. Visit the [Provide feedback about network protection](https://www.microsoft.com/wdsi/filesubmission/exploitguard/networkprotection) page if you want to report a website that could be a phishing site.
```

File: windows/security/threat-protection/microsoft-defender-atp/ios-install.md [hunk 15]
```diff
@@ -20,62 +20,118 @@ ms.collection:
...
-> 
-> As with any pre-release solution, remember to exercise caution when determining the target population for your deployments.
-
-Defender for Endpoint for iOS is currently available as a preview app on TestFlight, Apple's beta testing platform. In GA, it will be available on the Apple App store.
-
-Deployment devices need to be enrolled on Intune Company portal. Refer to
-[Enroll your
-device](https://docs.microsoft.com/mem/intune/enrollment/ios-enroll) to
-learn more about Intune device enrollment
+This topic describes deploying Defender for Endpoint for iOS on Intune Company Portal enrolled devices. For more information about Intune device enrollment, see [Enroll your device](https://docs.microsoft.com/mem/intune/enrollment/ios-enroll).
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/ios-install.md:27]:
```suggestion
This topic describes deploying Defender for Endpoint for iOS on Intune Company Portal enrolled devices. For more information about Intune device enrollment, see [Enroll iOS/iPadOS devices in Intune](https://docs.microsoft.com/mem/intune/enrollment/ios-enroll).
```

File: windows/security/threat-protection/microsoft-defender-atp/ios-install.md [hunk 16]
```diff
@@ -20,62 +20,118 @@ ms.collection:
...
 
--   Ensure you have access to [Microsoft Endpoint manager admin
-    center](https://go.microsoft.com/fwlink/?linkid=2109431).
+- Ensure you have access to [Microsoft Endpoint manager admin center](https://go.microsoft.com/fwlink/?linkid=2109431).
 
--   Ensure iOS enrollment is done for your users. Users need to have Defender for Endpoint
-    license assigned in order to use Defender for Endpoint for iOS. Refer [Assign licenses to
-    users](https://docs.microsoft.com/azure/active-directory/users-groups-roles/licensing-groups-assign)
-    for instructions on how to assign licenses.
+- Ensure iOS enrollment is done for your users. Users need to have Defender for Endpoint license assigned in order to use Defender for Endpoint for iOS. Refer [Assign licenses to users](https://docs.microsoft.com/azure/active-directory/users-groups-roles/licensing-groups-assign) for instructions on how to assign licenses.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/ios-install.md:33]:
```suggestion
- Ensure iOS enrollment is done for your users. Users need to have a Defender for Endpoint license assigned in order to use Defender for Endpoint for iOS. Refer to [Assign licenses to users](https://docs.microsoft.com/azure/active-directory/users-groups-roles/licensing-groups-assign) for instructions on how to assign licenses.
```

File: windows/security/threat-protection/microsoft-defender-atp/ios-install.md [hunk 17]
```diff
@@ -20,62 +20,118 @@ ms.collection:
...
 
+> [!NOTE]
+> **Microsoft Defender ATP (Microsoft Defender for Endpoint) for iOS is now available on [Apple App Store](https://aka.ms/mdatpiosappstore).**
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/ios-install.md:36]:
```suggestion
> Microsoft Defender ATP (Microsoft Defender for Endpoint) for iOS is now available on the [Apple App Store](https://aka.ms/mdatpiosappstore).
```

File: windows/security/threat-protection/microsoft-defender-atp/ios-install.md [hunk 18]
```diff
@@ -20,62 +20,118 @@ ms.collection:
...
 
-To install Defender for Endpoint for iOS, end-users can visit
-<https://aka.ms/defenderios> on their iOS devices. This link will open the
-TestFlight application on their device or prompt them to install TestFlight. On
-the TestFlight app, follow the onscreen instructions to install Defender for Endpoint.
+Deploy Defender for Endpoint for iOS via Intune Company Portal.
+
+### Add iOS store app
+
+1. In [Microsoft Endpoint manager admin center](https://go.microsoft.com/fwlink/?linkid=2109431), go to **Apps** -> **iOS/iPadOS** -> **Add** -> **iOS store app** and click Select.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/ios-install.md:44]:
```suggestion
1. In [Microsoft Endpoint manager admin center](https://go.microsoft.com/fwlink/?linkid=2109431), go to **Apps** -> **iOS/iPadOS** -> **Add** -> **iOS store app** and click **Select**.
```

File: windows/security/threat-protection/microsoft-defender-atp/ios-install.md [hunk 19]
```diff
@@ -20,62 +20,118 @@ ms.collection:
...
+
+1. Next, for enhanced Anti-phishing capabilities, you can deploy a custom profile on the supervised iOS devices. Follow the steps below:
+    - Download the config profile from [https://aka.ms/mdatpiossupervisedprofile](https://aka.ms/mdatpiossupervisedprofile)
+    - Navigate to **Devices** -> **iOS/iPadOS** -> **Configuration profiles** -> **Create Profile**
+
+    > [!div class="mx-imgBorder"]
+    ![Image of Microsoft Endpoint Manager Admin Center](images/ios-deploy-7.png)
+
+    - Provide a name of the profile. When prompted to import a Configuration profile file, select the one downloaded above.
+    - In the **Assignment** section, select the device group you want to apply this profile to. As a best practice, this should be applied for all managed iOS devices. Click **Next**
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/ios-install.md:132]:
```suggestion
    - In the **Assignment** section, select the device group to which you want to apply this profile. As a best practice, this should be applied to all managed iOS devices. Click **Next**.
```

File: windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios.md [hunk 20]
```diff
@@ -24,53 +24,51 @@ ms.topic: conceptual
...
 
+## Pre-requisites
 
 **For End Users**
 
--   Defender for Endpoint license assigned to the end user(s) of the app. Refer
-    [Assign licenses to
-    users](https://docs.microsoft.com/azure/active-directory/users-groups-roles/licensing-groups-assign)
-    for instructions on how to assign licenses.
+- Microsoft Defender for Endpoint license assigned to the end user(s) of the app. See [Microsoft Defender for Endpoint licensing requirements](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/minimum-requirements#licensing-requirements)
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios.md:37]:
```suggestion
- Microsoft Defender for Endpoint license assigned to the end user(s) of the app. See [Microsoft Defender for Endpoint licensing requirements](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/minimum-requirements#licensing-requirements).
```

File: windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios.md [hunk 21]
```diff
@@ -24,53 +24,51 @@ ms.topic: conceptual
...
--   Access to the Microsoft Defender Security Center portal
+- Access to the Microsoft Defender Security Center portal.
+
+    > [!NOTE]
+    > Microsoft Intune is the only supported Mobile Device Management (MDM) solution for deploying Microsoft Defender for Endpoint for iOS. Currently only enrolled devices are supported for enforcing Defender for Endpoint for iOS related device compliance policies in Intune.
 
--   Access to [Microsoft Endpoint Manager admin
-    center](https://go.microsoft.com/fwlink/?linkid=2109431), to deploy the app
-    to enrolled user groups in your organization
+- Access to [Microsoft Endpoint Manager admin center](https://go.microsoft.com/fwlink/?linkid=2109431), to deploy the app to enrolled user groups in your organization
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios.md:51]:
```suggestion
- Access to [Microsoft Endpoint Manager admin center](https://go.microsoft.com/fwlink/?linkid=2109431), to deploy the app to enrolled user groups in your organization.
```

File: windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios.md [hunk 22]
```diff
@@ -24,53 +24,51 @@ ms.topic: conceptual
...
 
 **System Requirements**
 
--   iOS devices running iOS 11.0 and later
+- iOS devices running iOS 11.0 and above.
 
--   Device is enrolled with Intune Company Portal
-    [app](https://apps.apple.com/us/app/intune-company-portal/id719171358)
+- Device is enrolled with Intune Company Portal [app](https://apps.apple.com/us/app/intune-company-portal/id719171358)
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios.md:57]:
```suggestion
- Device is enrolled with the [Intune Company Portalapp](https://apps.apple.com/us/app/intune-company-portal/id719171358).
```

File: windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios.md [hunk 23]
```diff
@@ -24,53 +24,51 @@ ms.topic: conceptual
...
+## Installation instructions
+
+Deployment of Microsoft Defender for Endpoint for iOS is via Microsoft Intune (MDM) and both supervised and unsupervised devices are supported.
+For more information, see [Deploy Microsoft Defender for Endpoint for iOS](ios-install.md).
+
+## Resources
 
--   Provide feedback through in-app feedback system or through [SecOps
-    portal](https://securitycenter.microsoft.com)
+- Stay informed about upcoming releases by visiting our [blog](https://techcommunity.microsoft.com/t5/microsoft-defender-atp/bg-p/MicrosoftDefenderATPBlog/label-name/iOS)
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios.md:69]:
```suggestion
- Stay informed about upcoming releases by visiting our [blog](https://techcommunity.microsoft.com/t5/microsoft-defender-atp/bg-p/MicrosoftDefenderATPBlog/label-name/iOS).
```

File: windows/security/threat-protection/microsoft-defender-atp/ios-install.md [hunk 24]
```diff
@@ -20,62 +20,118 @@ ms.collection:
...
-3.  Upon successful onboarding, the device will start showing up on the Devices
-    list in Microsoft Defender Security Center.
+3. Upon successful onboarding, the device will start showing up on the Devices list in Microsoft Defender Security Center.
 
     > [!div class="mx-imgBorder"]
     > ![A screenshot of a cell phone Description automatically generated](images/e07f270419f7b1e5ee6744f8b38ddeaf.png)
 
+## Configure Microsoft Defender for Endpoint for Supervised Mode
+
+The Microsoft Defender for Endpoint for iOS app has specialized ability on supervised iOS/iPadOS devices, given the increased management capabilities provided by the platform on these types of devices. To take advantage of these capabilities, the Defender for Endpoint app needs to know if a device is in Supervised mode.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/ios-install.md:84]:
```suggestion
The Microsoft Defender for Endpoint for iOS app has specialized ability on supervised iOS/iPadOS devices, given the increased management capabilities provided by the platform on these types of devices. To take advantage of these capabilities, the Defender for Endpoint app needs to know if a device is in Supervised Mode.
```

File: windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios.md [hunk 25]
```diff
@@ -24,53 +24,51 @@ ms.topic: conceptual
...
 **For End Users**
 
--   Defender for Endpoint license assigned to the end user(s) of the app. Refer
-    [Assign licenses to
-    users](https://docs.microsoft.com/azure/active-directory/users-groups-roles/licensing-groups-assign)
-    for instructions on how to assign licenses.
+- Microsoft Defender for Endpoint license assigned to the end user(s) of the app. See [Microsoft Defender for Endpoint licensing requirements](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/minimum-requirements#licensing-requirements)
+
+- Device(s) are [enrolled](https://docs.microsoft.com/mem/intune/user-help/enroll-your-device-in-intune-ios) via the Intune Company Portal app to enforce Intune device compliance policies. This requires the end user to be assigned a Microsoft Intune license.
+    - Intune Company Portal app can be downloaded from [Apple App Store](https://apps.apple.com/us/app/intune-company-portal/id719171358).
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios.md:40]:
```suggestion
    - Intune Company Portal app can be downloaded from the [Apple App Store](https://apps.apple.com/us/app/intune-company-portal/id719171358).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios.md (hunk 22 above)
Comment (illfated) [reply windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios.md:57]: @JohanFreelancer9 : Are you sure about this? The word "Portalapp" looks unusual to me, but maybe I am too familiar with British English and the norm of not combining words.

**Comment by illfated** [comment]
Thank you for the effort. Good job. I notice I missed a lot of details.

File: windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios.md (hunk 22 above)
Comment (JohanFreelancer9) [reply windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios.md]: @illfated - my bad. Thank you for noticing.

**Review by JohanFreelancer9 — APPROVED** [review]

File: windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios.md (hunk 11 above)
Comment (illfated) [reply windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-atp-ios.md:27]: That is very OK. Don't worry. I just wanted to make sure I hadn't missed something and needed to confirm this phrase. 🙂

**Review by illfated — APPROVED** [review]

**Comment by illfated** [comment]
#sign-off

**Comment by sunasing** [comment]
#sign-off

**Comment by shadows911** [comment]
> #sign-off

not so good

**Pull request merged**